ui/components/app/edit-gas-display/index.scss

```diff
@@ -39,35 +26,10 @@
     margin: 20px auto;
   }
 
-  &__advanced-button {
```

Comment:
No classname exists for these styles
![Screenshot 2023-07-28 at 3 54 46 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/86eeb862-d216-4590-91cc-61c709d1aabe)